[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-1 (#114341)
openshift/openshift-docs: modules/storage-persistent-storage-aws-maximum-volumes.adoc

[id="maximum-number-of-ebs-volumes-on-a-node_{context}"]
= Maximum number of EBS volumes on a node

[role="_abstract"]

> By default, {product-title} supports a maximum of 39 EBS volumes attached to one node. This limit is consistent with the AWS volume limits. The volume limit depends on the instance type.

Reviewer: ```suggestion
By default, you can attach a  maximum of 39 EBS volumes one node. This limit is consistent with the AWS volume limits. The volume limit depends on the instance type.
```
